azure-pipelines/WIFtoPATauth.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+- name: deadPATServiceConnectionId # The GUID of the PAT-based service connection whose access token must be replaced.
+  type: string
+- name: wifServiceConnectionName # The name of the WIF service connection to use to get the access token.
+  type: string
+- name: resource # The scope for which the access token is requested.
+  type: string
+  default: 499b84ac-1321-427f-aa17-267ca6975798 # Azure Artifact feeds (any of them)
+
+steps:
+- task: AzureCLI@2
+  displayName: 🔏 Authenticate with WIF service connection
```

Comment:
Is the emoji required? I find them very noisy in the AzDO build pages and wish there were fewer injected by the 1ES template stuff.

Reply:
It's not required, but it seems appropriate due to this being a 1ES thing (they all have emoji) and not something custom for us. I regret to inform you that 80% tasks in the pipeline are 1ES injected with emoji included.